package.json

```diff
@@ -69,13 +69,13 @@
         "eleventy-plugin-nesting-toc": "^1.3.0",
         "eleventy-plugin-page-assets": "^0.3.0",
         "eleventy-plugin-reading-time": "^0.0.1",
-        "eslint": "^8.19.0",
-        "eslint-config-eslint": "^7.0.0",
-        "eslint-plugin-jsdoc": "^39.2.0",
-        "eslint-plugin-jsx-a11y": "^6.5.1",
+        "eslint": "^8.56.0",
+        "eslint-config-eslint": "^9.0.0",
+        "eslint-plugin-jsx-a11y": "^6.8.0",
         "eslint-plugin-node": "^11.1.0",
```

Comment:
```suggestion
```

We can now remove this.

Note: package-lock.json should probably also be updated after this is removed.

Reply:
Yes, removed!